plugins/config/src/configenv/configenv.go

```diff
@@ -0,0 +1,194 @@
+package configenv
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"archive/tar"
+
+	common "github.com/dokku/dokku/plugins/common"
+)
+
+//Env is a representation for global or app environment
+type Env struct {
+	name           string
+	env            map[string]string
+	filename       string
+	EscapeNewlines bool
+}
+
+func (e *Env) String() string {
+	return e.EnvfileString()
+}
+
+//EnvfileString returns the contents of this Env in ENVFILE format
+func (e *Env) EnvfileString() string {
```

Comment:
I like that idea. Right now they only support reading, not writing. I'll see if they are interested in a PR for writing out .env files too. Even if not I agree it's worth the move.